changelogs/unreleased/gh-8744-fix-chunked-with-content-length.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/lua/http client
```

Comment:
How about reporting the problem against the libcurl's documentation for `CURLOPT_UPLOAD`?

Reply:
Merged: https://github.com/curl/curl/pull/11300

Reply:
Thanks! It obsoletes the commit/PR description a bit: I think we should highlight that the quoted option description is updated now. Otherwise it may be confusing.

Reply:
Done.